Rakefile

```diff
@@ -124,8 +124,24 @@ task :git_checks do
   abort red 'Git branch is not up to date please pull' unless branch_up_to_date
 end
 
-task ci: [:erase_devices, :all_checks]
-task all_checks: [:lint, :analyze, :test]
+task ci: [:erase_devices, :lint, :ci_uikit, :ci_swiftui]
+
+task :ci_uikit do
+  task(:all_checks).invoke(EXAMPLE_SCHEME)
+  task(:all_checks).reenable
+end
+
+task :ci_swiftui do
+  task(:all_checks).invoke(SWIFTUI_SCHEME)
+  task(:all_checks).reenable
+end
```

Comment:
because ruby and the Rakefile are weird, its much easier to have 2 tasks that send parameters like this, instead of having a single ci task and have the param being sent by the github action yml.